docs/cookbook.html

```diff
@@ -450,13 +450,13 @@ <h2 id="_overview">Overview</h2>
 <div class="sectionbody">
 <div class="paragraph">
 <p>The following are some recipes to address issues which may arise while using
-Brine but are not considered part of Brine&#8217;s responsibility (at least presently).
-Many of these are focused on simplicity and ease rather than robustness or elegance,
-and modification to address specific cases should be expected.</p>
+Brine but are not considered part of Brine&#8217;s (current) responsibility.
+Many of these are focused on simplicity and ease rather than robustness or
+elegance, and modification to address specific cases should be expected.</p>
```

Comment:
A bunch of column 80 filling so it's more reliably easy to work with.

Reply:
Actually these are all generated anyway, but the source files had the fill corrected.